### PR TITLE
No longer rebuild every single time on OSX

### PR DESCRIPTION
**Pull request by helenol**
Turns out this CMake was an unnecessary mess.

@mfehr 
And FYI @ffurrer since you have the same issue. :)

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -10,9 +10,9 @@ file(MAKE_DIRECTORY ${CATKIN_DEVEL_PREFIX}/include)
 
 set(VERSION 0.3.4)
 ExternalProject_Add(glog_src
-  DOWNLOAD_COMMAND wget --retry-connrefused --waitretry=1 --timeout=12 --tries 10 https://github.com/google/glog/archive/v${VERSION}.zip
```
Comment (mfehr) [inline CMakeLists.txt]: So this will drop all the robustness against connection loss?

Comment (mfehr) [inline CMakeLists.txt]: @helenol are you sure the DOWNLOAD_COMMAND was the source of the rebuild and not the UPDATE_COMMAND?

Comment (mfehr) [inline CMakeLists.txt]: Discussed offline. Seems ok.

**Comment by mfehr** [comment]
@schneith Since you introduced the retries into the CMakeLists.txt, could you have a quick look?

**Comment by schneith** [comment]
We might need to re-introduce the download retries if the connection is still unreliable. But let's try it without them for now.

**Comment by helenol** [comment]
@schneith most of our other catkin wrappers (gflags, ceres, etc.) don't have a manual download command. Also CMake has 3 retried by default, as far as I can tell: http://www.ccp4.ac.uk/dist/checkout/cmake-3.1.3/Modules/ExternalProject.cmake set(retries 3) for URL entries

But yeah if this poses problems we should look at this again, @mfehr suggested the unzip command is actually the problem for OS X, so we could just bundle the download step with unzipping and leave update command blank

**Comment by schneith** [comment]
Cmake 2.8.12.2 on Ubuntu 14.04 does not support the retries by default. But instead of patching all the projects we could just update all the build servers. That might be easier :)

**Pull request merged**